### PR TITLE
Persona API Support

### PR DESCRIPTION
**Pull request by carlosclayton**
Fixes #[include a number of issue this PR is fixing].

Summary of changes proposed in this Pull Request:
- 
- 
- 

PR checklist:
- [ ] Updated CHANGELOG.md

### CODE REVIEW

**Comment by carlosclayton** [comment]
Support to send message using Persona API

**Comment by wittfabian** [comment]
https://developers.facebook.com/docs/messenger-platform/send-messages/personas/

File: Messages/SenderAction.php [hunk 1]
```diff
@@ -52,7 +58,8 @@ public function getData()
             'recipient' =>  [
                 'id' => $this->recipient
             ],
-            'sender_action' => $this->action
+            'sender_action' => $this->action,
+            'persona_id' => $this->persona
         ];
     }
 }
```
Comment (wittfabian) [inline Messages/SenderAction.php:65]: No newline at end of file!

**Comment by wittfabian** [comment]
* Rebase your code to the latest version
* Provide documentation for the changes
* Update your description and the CHANGELOG

**Pull request closed without merging**